src/Components/Config/ConfigScreens/ServerScreen.js

```diff
@@ -187,6 +187,31 @@ class ServerScreen extends Component {
             </div>
           </section>
         </OnlyIf>
+
+        <OnlyIf test={this.state.automine}>
+          <section>
+            <h4>ERROR ON TRANSACTION FAILURE</h4>
+            <div className="Row">
+              <div className="RowItem">
+                <div className="Switch">
+                  <input
+                    type="checkbox"
+                    name="server.vmErrorsOnRPCResponse"
+                    id="server.vmErrorsOnRPCResponse"
+                    defaultChecked={this.props.settings.server.vmErrorsOnRPCResponse}
+                    onChange={this.props.handleInputChange}
+                  />
+                  <label htmlFor="server.vmErrorsOnRPCResponse">ENABLED</label>
+                </div>
+              </div>
+              <div className="RowItem">
+                <p>
+                  When transactions fail, return an RPC error. If disabled, transaction failures will only be detectable via the "status" flag in the transaction receipt. Disabling this is recommended for newer projects. This will eventually be the default.
```

Comment:
Can you change this copy to the following? (Happy to discuss different wording, etc.) 

> When transactions fail, throw an error. If disabled, transaction failures will only be detectable via the "status" flag in the transaction receipt. Disabling this feature will make Ganache handle transaction failures like other Ethereum clients. 

Basically, I don't think we should be recommending this be disabled as none of our other tooling handles status message errors properly. We should recommend this feature be turned off when the changes to `truffle-contract` have become widely used.

Reply:
> We should recommend this feature be turned off when the changes to truffle-contract have become widely used.

:+1: happy w/ that approach.

You mentioned elsewhere a request to move this to the advanced page. I'd reply there, but I can't find it anymore. I can if you want me to, but putting it here had the benefit of communicating that this only applies when instamining (that is, it's hidden when interval mining).